go/host/storage/hostdb/rollup.go

```diff
@@ -14,8 +14,10 @@ import (
 )
 
 const (
-	selectExtRollup    = "SELECT ext_rollup from rollup_host r"
-	selectLatestRollup = "SELECT ext_rollup FROM rollup_host ORDER BY time_stamp DESC LIMIT 1"
+	selectExtRollup     = "SELECT ext_rollup from rollup_host r"
+	selectLatestRollup  = "SELECT ext_rollup FROM rollup_host ORDER BY time_stamp DESC LIMIT 1"
+	selectRollupBatches = "SELECT b.sequence, b.hash, b.full_hash, b.height, b.ext_batch FROM rollup_host r JOIN batch_host b ON r.start_seq <= b.sequence AND r.end_seq >= b.sequence"
```

Comment:
I think it would be good to `order by b.height desc`

Reply:
Good spot, made the change. Cheers